core/src/main/kotlin/org/evomaster/core/search/service/monitor/SearchOverall.kt

```diff
@@ -0,0 +1,41 @@
+package org.evomaster.core.search.service
+
+import org.evomaster.core.problem.rest.RestCallAction
+import org.evomaster.core.search.Action
+import org.evomaster.core.search.EvaluatedIndividual
+import org.evomaster.core.search.Individual
+
+
+/**
+ * @author: manzhang
```

Comment:
can u add some comments here on what this calls is for?

Reply:
The comment is added.